src/app/components/modal/index.html

```diff
@@ -179,6 +168,25 @@
 				<li>The form supports a complex task for a list of objects such as filling out report cards for a class.</li>
 				<li>The task includes subtasks that need to be completed in context.</li>
 			</ul>
+    
+		<div class="sky-section-heading">Content</div>
+		<div class="sky-subsection-heading">Modal title text</div>
+			<p>In general, modal titles should use sentence-case capitalization and the following format:</p>
+			<p><span class="sky-emphasized">&lt;Verb&gt; &lt;direct object&gt; for &lt;indirect object&gt;</span></p>
+			<div style="margin-bottom: 20px;">
+				<img class="screenshot screenshot-info" src="~/assets/img/guidelines/wording/modal-with-object.png" alt="modal title: Edit address for Robert Hernandez"/>
+			</div>
+			<p>If there is not an indirect object or the user is the implied indirect object, you can exclude it:</p>
+			<p><span class="sky-emphasized">&lt;Verb&gt; &lt;direct object&gt;</span></p>
+			<div style="margin-bottom: 20px;">
+				<img class="screenshot screenshot-info" src="~/assets/img/guidelines/wording/modal-without-object.png" alt="modal title: Manage bank feeds"/>
+			</div>
+		
+		<div class="sky-subsection-heading">Buttons in the modal action bar</div>
+			<ul>
+				<li>Use a primary <a routerLink="/components/button">button</a> for the most likely action the user will take. Use secondary buttons for less likely actions. Use a link button for the Cancel action.</li>
```

Comment:
Might just be me, but I think "most likely action" reads awkwardly. I'd change this to "... for the most important action. Use secondary buttons for less important actions..."

Reply:
that sounds good to me.

Reply:
✔️